index.js

```diff
@@ -51,6 +52,9 @@ var Omegle = function () {
 		}
 		request(options, function (error, response, body) {
 			if (!error && response.statusCode == 200) {
+				if(!working_servers.indexOf(url)>0){
```

Comment:
Hm, I will look into that and then we can do it in another PR. For now, let's just fix the condition and variable name and then I'll merge your commits. 👍